### ISSUE TITLE
Split Large Monitor into Multiple Logical Screens

### ISSUE THREAD

**Issue by surmeier** [labwc/labwc#903]
Similar to this [Mutter issue](https://gitlab.gnome.org/GNOME/mutter/-/issues/1507), it would be nice to be able to split large monitors into smaller logical screens. This is especially useful when using a TV or ultra-wide monitor as a main display. I'm not familiar with how Microsoft does this with Fancy Zones, but the solution would implement something similar. 

If this is a feature people want, this is something I could work on and make a pull request for.

**Comment by Consolatis** [comment]
We have regions that could be used to partition the screen. We'd need some way for the `Maximize` button to use the current region though to make it feel more natural. Are there other missing features?

**Comment by surmeier** [comment]
I agree that app window confinement to some arbitrary region in the screen is most of the work. 

It would be ideal for any window action to respect the limited region, i.e not only Maximize but any MoveToEdge/SnapToEdge command as well. 

A different solution is treating the region more like a fully emulated monitor, though obviously it won't have all the features of an actual physical monitor. I'm not sure if this would be worth it, or what benefits this would have with the downside of increased complexity.

**Comment by johanmalm** [comment]
You could do the following:

1. Run a nested instance of labwc with WLR_WL_OUTPUTS=2
2. Disable serverDecoration using the new window-actions in PR #901 
3. Arrange the two labwc windows side-by-side with regions or SnapToEdge
4. Run ToggleKeybinds action to pass key binds to the nested instance.

**Comment by Consolatis** [comment]
That would be a solution as well and would in fact mean two separate outputs. And the performance shouldn't be as bad as it sounds either. Alternative to step 2: Manually using the `ToggleDecorations` action (or `Decorations` in the window menu).

A downside is moving windows between the 2 outputs is a bit annoying, e.g. you can't start a move operation on one output and then move it over to the other.
